.github/workflows/arduino_test_runner.yml

```diff
@@ -1,17 +1,34 @@
----
 name: Arduino CI
 
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - main
```

Comment:
main branch should be master branch

Reply:
Good spot, I'll change it. Maybe we should migrate to the new naming standard at some point - no that it actually makes a material difference